doc/articles/getting-started/GettingStartedWithTests.md

```diff
@@ -0,0 +1,61 @@
+---
+uid: Uno.Authoring.Tests
+---
+# How-To: Creating an application with tests using Uno.Extensions
+ 
+This tutorial will walk through how to create an Uno application with the `dotnet new` tool, which is already configured to use the Uno.Extensions.
```

Comment:
Instead of saying that this will walk people through "how to create an Uno app ... " should we point people to how to create an app with Tests? 
https://platform.uno/docs/articles/getting-started/wizard/using-wizard.html#10-testing

cc @agneszitte @vatsashah45 @dansiegel

Reply:
Yes, we need instructions or pointer to how to create with tests.